src/BlackSlope.Api.Common/BlackSlope.Api.Common.csproj

```diff
@@ -1,8 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
-    <LangVersion>9.0</LangVersion>

```

Comment:
No reason for us to keep targeting 9.x. We ought default to the latest language impl unless we have a real good reason.